frontend/src/pages/projects/screens/projects/NewProjectButton.tsx

```diff
@@ -24,11 +22,6 @@ const NewProjectButton: React.FC<NewProjectButtonProps> = ({ closeOnCreate, onPr
       <ManageProjectModal
         open={open}
         onClose={(newProjectName) => {
-          fireTrackingEvent('NewProject Created', {
```

Comment:
Why did you drop this?

Reply:
This is now handled inside the ManageProjectModal at https://github.com/opendatahub-io/odh-dashboard/pull/2781/files#diff-8827454ead4ff8421767576741c1ec341bf780e07671a2c37a34fd2677fbb7caR51 , because the Modal is called from multiple places.

Reply:
@pilhuhn there are 5 use cases of `NewProjectButton` -- you point at solving 1. What about the other 4?

Reply:
I only ask because we sound like we are losing tracking for the home page

Reply:
@andrewballantyne I am not sure I understand your question. The tracking code has moved from `NewProjectButton` into `ManageProjectModal`, which is part of the internal implementation of the `NewProjectButton`.
I did that because the `ProjectsSection` is using the modal directly and not via `NewProjectButton`.

Reply:
I misunderstood it when I briefly was looking at it earlier. Thanks for the clarification.